common/api/core-frontend.api.md

```diff
@@ -8115,11 +8146,34 @@ export namespace RealityDataSource {
     export function createOrbitGtBlobPropsFromKey(rdSourceKey: RealityDataSourceKey): OrbitGtBlobProps | undefined;
     // @alpha
     export function fromKey(key: RealityDataSourceKey, iTwinId: GuidString | undefined): Promise<RealityDataSource | undefined>;
+    // @alpha
+    export function setBaseUrl(id: string): void;
+}
+
+// @alpha
+export class RealityDataSourceGP3DTProvider implements RealityDataSourceProvider {

```

Comment:
Let's make a NextVersion.md entry with content similar to the description of this PR:
- How to use new RealityDataSourceGP3DTProvider to display GP3DT tiles
- Make sure any included screenshots are small enough, jpeg, and include proper attribution in the viewable portion (bottom left).

Reply:
Pushed the version we discussed, with the logos easier to see in the screenshots.

Reply:
Looks good, thanks.